SQLQuery5.sql

```diff
@@ -0,0 +1,5 @@
+﻿SELECT  Employee.Id AS 'Id', FirstName, 
+LastName, Department.Name AS 'DeptName', isSupervisor ,  Computer.Make, Computer.Manufacturer
```

Comment:
Did you mean to push up this file as well? If you have  SQL queries that you're testing, you don't need to save them and commit them. If they're queries that other people need, you should save them with clear naming conventions and describe what they do in your PR